### PR TITLE
use uuid.valuse instead of dump.values

### PR DESCRIPTION
**Pull request by chichinohaha**
Re: cocos-creator/3d-tasks#

Changelog:
 * 

-->

### CODE REVIEW

**Review by VisualSJ — APPROVED** [review]

File: editor/inspector/components/batched-skinning-model.js [hunk 1]
```diff
@@ -20,22 +20,19 @@ exports.ready = function () {
     $prop.appendChild($button);
 
     $button.addEventListener('confirm', () => {
-        let values = [this.dump.value];
-        if (this.dump.values) {
-            values = this.dump.values;
-        }
+        let uuids = this.dump.value.uuid.values || [this.dump.value.uuid.value];
```
Comment (arsen2010) [inline editor/inspector/components/batched-skinning-model.js]: let 改为 const

File: editor/inspector/components/polygon-collider.js [hunk 2]
```diff
@@ -20,13 +20,10 @@ exports.ready = function () {
                 $input.after($button);
 
                 $button.addEventListener('confirm', () => {
-                    let values = [this.dump.value];
-                    if (this.dump.values) {
-                        values = this.dump.values;
-                    }
+                    let uuids = this.dump.value.uuid.values || [this.dump.value.uuid.value];
```
Comment (arsen2010) [inline editor/inspector/components/polygon-collider.js]: let 改为 const

File: editor/inspector/components/batched-skinning-model.js (hunk 1 above)
Comment (chichinohaha) [reply editor/inspector/components/batched-skinning-model.js]: 已修改

File: editor/inspector/components/polygon-collider.js (hunk 2 above)
Comment (chichinohaha) [reply editor/inspector/components/polygon-collider.js]: done

**Review by arsen2010 — APPROVED** [review]

**Pull request merged**